layouts/404/single.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="p404-page page">
+    {{ partial "banner" .}}
+    <div  class="container-fluid">
+        <div class="container">
```

Comment:
Same here, no need for this one also.

Reply:
It messes up the alignment
<img width="1133" alt="logo" src="https://user-images.githubusercontent.com/3217669/81724054-09d88480-9484-11ea-811c-e2226751b22e.png">

Reply:
thanks for pointing these out!